tests/models/test_serialized_dag.py

```diff
@@ -63,60 +72,68 @@ def _write_example_dags(self):
         return example_dags
 
     def test_write_dag(self):
-        """DAGs can be written into database."""
-        example_dags = self._write_example_dags()
+        """DAGs can be written into database when no comprehension"""
+        with mock.patch(
+            'airflow.models.serialized_dag.COMPRESS_SERIALIZED_DAGS', self.compress_serialized_dags
+        ):
```

Comment:
Rather than having to do this and touch most of the lines of the files you can do this:

```python
    def setUp(self):
        self.patcher = mock.patch(
            'airflow.models.serialized_dag.COMPRESS_SERIALIZED_DAGS', self.compress_serialized_dags
        )
        self.patcher.start()

    def tearDown(sefl):
        self.patcher.stop()
```